src/main/java/com/ecommerce/domain/customer/Customer.java

```diff
@@ -0,0 +1,86 @@
+package com.ecommerce.domain.customer;
+
+import com.ecommerce.domain.BaseTimeEntity;
+import lombok.*;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Entity
+@Getter @Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class Customer extends BaseTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "customer_id")
+    private Long id;
+
+    @Column(nullable = false, length = 30,unique = true)
+    private String username;
+
+    private String password;
+
+    @Column(nullable = false, length = 30)
+    private String name;
+
+    @Column(nullable = false, length = 30)
+    private String phone;
+
+    @Column(nullable = false, length = 30)
+    private String nickName;
+
+    @Column(nullable = false, length = 30)
+    private String address;
+
+    @Column(nullable = false, length = 30)
+    private Integer age;
+
+    private String roles;
+
+    public List<String> getRoleList(){
+        if (this.roles.length()>0){
```

Comment:
```suggestion
        if (this.roles.length() > 0) {
```
으로 해주시는게 java coding convention 에 부합하는 코드입니다~!

아래 링크를 통해 한번 보시구 조금씩 적용해보시면 좋겠네요 ^^;
https://naver.github.io/hackday-conventions-java/